README.md

```diff
@@ -1 +1,32 @@
-# deephaven-riptable
+# Deephaven Riptable Integration
+
+deephaven-riptable is a Python package created by Deephaven Data Labs. It supports the conversion between Riptable Datasets and Deephaven tables.
```

Comment:
and vice versa?

Reply:
Does `between' imply bi-directional?

Reply:
The way it is worded implied unidirectional to me.